File: Color Code Cipher Locked/Java/Encoder.java
import java.util.*;

public class Encoder {
    public static int[] qwertyuiopasdfghjklzxcvbnm(String mnbvcxzlkjhgfdsapoiuytrewq) {
        int[] asdfghjklqwertyuiop = new int[mnbvcxzlkjhgfdsapoiuytrewq.length()];
        
        for(int qwertyasdfgh = 0; qwertyasdfgh < mnbvcxzlkjhgfdsapoiuytrewq.length(); qwertyasdfgh++) {
            char zxcvbnmlkjhgfdsa = mnbvcxzlkjhgfdsapoiuytrewq.charAt(qwertyasdfgh);
            int qazwsxedcrfvtgbyhnujmikolp = zxcvbnmlkjhgfdsa - 'A';
            asdfghjklqwertyuiop[qwertyasdfgh] = qazwsxedcrfvtgbyhnujmikolp;
        }
        
        return asdfghjklqwertyuiop;
    }
    
    public static int[] plmoknjibvhugcyfxtdrzeaswq(int[] yhgtfrdeswaqqazxswedcvfrtgbnhyujm) {
        int[] rfvbgtyhnmjuiklopmqazwsxedc = new int[yhgtfrdeswaqqazxswedcvfrtgbnhyujm.length];
        
        for(int qwertyuiop = 0; qwertyuiop < yhgtfrdeswaqqazxswedcvfrtgbnhyujm.length; qwertyuiop++) {
            rfvbgtyhnmjuiklopmqazwsxedc[qwertyuiop] = yhgtfrdeswaqqazxswedcvfrtgbnhyujm[yhgtfrdeswaqqazxswedcvfrtgbnhyujm.length - 1 - qwertyuiop];
        }
        
        return rfvbgtyhnmjuiklopmqazwsxedc;
    }
    
    public static int[] zaqxswcdevfrbgtnhymjukilop(int[] wqertyuiopasdfghjklzxcvbnm) {
        int[] qscwdvefbrgtnhymjuik = new int[wqertyuiopasdfghjklzxcvbnm.length];
        
        for(int plokimjunhybgtvfrcdxeszwaq = 0; plokimjunhybgtvfrcdxeszwaq < wqertyuiopasdfghjklzxcvbnm.length; plokimjunhybgtvfrcdxeszwaq++) {
            int mnhbgvfcdxsza = wqertyuiopasdfghjklzxcvbnm[plokimjunhybgtvfrcdxeszwaq] * (plokimjunhybgtvfrcdxeszwaq + 1);
            qscwdvefbrgtnhymjuik[plokimjunhybgtvfrcdxeszwaq] = mnhbgvfcdxsza;
        }
        
        return qscwdvefbrgtnhymjuik;
    }
    
    public static void polikujmyhntgbrfvedcwsxzaq() {
        Scanner ____________________________________________ = new Scanner(System.in);
        System.out.print("Enter string to encode: ");
        String qwaszxerdfcvtyghbnujikolpm = ____________________________________________.nextLine();
        
        int[] xswedrftgyhujikolpmnbvcxz = qwertyuiopasdfghjklzxcvbnm(qwaszxerdfcvtyghbnujikolpm);
        int[] cxdseawqrfvbgtyhnujmikol = plmoknjibvhugcyfxtdrzeaswq(xswedrftgyhujikolpmnbvcxz);
        int[] qazxswedcvfrbgtnhyjumkilop = zaqxswcdevfrbgtnhymjukilop(cxdseawqrfvbgtyhnujmikol);
        
        System.out.print("Encoded: ");
        for(int bhgfvcdxszaqwerty = 0; bhgfvcdxszaqwerty < qazxswedcvfrbgtnhyjumkilop.length; bhgfvcdxszaqwerty++) {
            System.out.print(qazxswedcvfrbgtnhyjumkilop[bhgfvcdxszaqwerty]);
            if(bhgfvcdxszaqwerty < qazxswedcvfrbgtnhyjumkilop.length - 1) 
                System.out.print(" ");
        }
        System.out.println();
        
        ____________________________________________.close();
    }
    
    public static void main(String[] args) {
        polikujmyhntgbrfvedcwsxzaq();
    }
}

File: Color Code Cipher Locked/C/encoder.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

#define _________________________________________________________________________ int*
#define _____________________________________________________________ char*
#define ________________________________________________________ int
#define ________________________________________________ printf
#define ____________________________________________ scanf
#define _______________________________________ "\n"
#define _________________________________ malloc
#define ____________________________ strlen
#define _________________________ free

_________________________________________________________________________  qwertyuiopasdfghjklzxcvbnm(_____________________________________________________________ mnbvcxzlkjhgfdsapoiuytrewq, ________________________________________________________ *plmnjhbgvfcdxsza) {
    ________________________________________________________ zxcvbnmlkjhgfdsaqwerty = ____________________________(mnbvcxzlkjhgfdsapoiuytrewq);
    _________________________________________________________________________ asdfghjklqwertyuiop = (________________________________________________________ *)_________________________________(zxcvbnmlkjhgfdsaqwerty * sizeof(________________________________________________________));
    
    for(________________________________________________________ qwertyasdfgh = 0; qwertyasdfgh < zxcvbnmlkjhgfdsaqwerty; qwertyasdfgh++) {
        ________________________________________________________ qazwsxedcrfvtgbyhnujmikolp = mnbvcxzlkjhgfdsapoiuytrewq[qwertyasdfgh] - 'A';
        asdfghjklqwertyuiop[qwertyasdfgh] = qazwsxedcrfvtgbyhnujmikolp;
    }
    
    *plmnjhbgvfcdxsza = zxcvbnmlkjhgfdsaqwerty;
    return asdfghjklqwertyuiop;
}

_________________________________________________________________________ plmoknjibvhugcyfxtdrzeaswq(_________________________________________________________________________ yhgtfrdeswaqqazxswedcvfrtgbnhyujm, ________________________________________________________ mnbvcxzaqwerty) {
    _________________________________________________________________________ rfvbgtyhnmjuiklopmqazwsxedc = (________________________________________________________ *)_________________________________(mnbvcxzaqwerty * sizeof(________________________________________________________));
    
    for(________________________________________________________ qwertyuiop = 0; qwertyuiop < mnbvcxzaqwerty; qwertyuiop++) {
        rfvbgtyhnmjuiklopmqazwsxedc[qwertyuiop] = yhgtfrdeswaqqazxswedcvfrtgbnhyujm[mnbvcxzaqwerty - 1 - qwertyuiop];
    }
    
    return rfvbgtyhnmjuiklopmqazwsxedc;
}

_________________________________________________________________________ zaqxswcdevfrbgtnhymjukilop(_________________________________________________________________________ wqertyuiopasdfghjklzxcvbnm, ________________________________________________________ asdfghjklzxc) {
    _________________________________________________________________________ qscwdvefbrgtnhymjuik = (________________________________________________________ *)_________________________________(asdfghjklzxc * sizeof(________________________________________________________));
    
    for(________________________________________________________ plokimjunhybgtvfrcdxeszwaq = 0; plokimjunhybgtvfrcdxeszwaq < asdfghjklzxc; plokimjunhybgtvfrcdxeszwaq++) {
        ________________________________________________________ mnhbgvfcdxsza = wqertyuiopasdfghjklzxcvbnm[plokimjunhybgtvfrcdxeszwaq] * (plokimjunhybgtvfrcdxeszwaq + 1);
        qscwdvefbrgtnhymjuik[plokimjunhybgtvfrcdxeszwaq] = mnhbgvfcdxsza;
    }
    
    return qscwdvefbrgtnhymjuik;
}

void polikujmyhntgbrfvedcwsxzaq() {
    _____________________________________________________________ qwaszxerdfcvtyghbnujikolpm = (_____________________________________________________________ )_________________________________(100 * sizeof(char));
    ________________________________________________("Enter string to encode: ");
    ____________________________________________("%s", qwaszxerdfcvtyghbnujikolpm);
    
    ________________________________________________________ qwertyuioplength;
    _________________________________________________________________________ xswedrftgyhujikolpmnbvcxz = qwertyuiopasdfghjklzxcvbnm(qwaszxerdfcvtyghbnujikolpm, &qwertyuioplength);
    _________________________________________________________________________ cxdseawqrfvbgtyhnujmikol = plmoknjibvhugcyfxtdrzeaswq(xswedrftgyhujikolpmnbvcxz, qwertyuioplength);
    _________________________________________________________________________ qazxswedcvfrbgtnhyjumkilop = zaqxswcdevfrbgtnhymjukilop(cxdseawqrfvbgtyhnujmikol, qwertyuioplength);
    
    ________________________________________________("Encoded: ");
    for(________________________________________________________ bhgfvcdxszaqwerty = 0; bhgfvcdxszaqwerty < qwertyuioplength; bhgfvcdxszaqwerty++) {
        ________________________________________________("%d", qazxswedcvfrbgtnhyjumkilop[bhgfvcdxszaqwerty]);
        if(bhgfvcdxszaqwerty < qwertyuioplength - 1) 
            ________________________________________________(" ");
    }
    ________________________________________________(_______________________________________);
    
    _________________________(qwaszxerdfcvtyghbnujikolpm);
    _________________________(xswedrftgyhujikolpmnbvcxz);
    _________________________(cxdseawqrfvbgtyhnujmikol);
    _________________________(qazxswedcvfrbgtnhyjumkilop);
}

________________________________________________________ main() {
    polikujmyhntgbrfvedcwsxzaq();
    return 0;
}